docs/src/pages/components/theme.mdx

```diff
@@ -0,0 +1,56 @@
+---
+name: Theme
+route: /components/theme
+---
+
+import Theme from '@pluralsight/ps-design-system-theme'
+
+import { Types, Usage } from '../../components/component'
+
+# Theme
+
+## Usage pattern
```

Comment:
```suggestion
## Examples
```

Not sure this is necessarily required.  It's just the heading that we've used as a standard so far.